BluetoothLEExplorer/BluetoothLEExplorer/Models/GattSampleContext.cs

```diff
@@ -337,6 +337,38 @@ private string GetDelimitedAddr(string addr)
             return string.Join(":", list);
         }
 
+        /// <summary>
+        /// Clears all devices
+        /// </summary>
+        public void ClearAllDevices()
+        {
+            try

```

Comment:
Use members vs properties for internal access, if possible reduce properties to the public contract, for all similar patterns in this review please.